packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 5.4.1 - Feb 07, 2024
+Hotfix version.
+
+We made UI and docs updates to multiple places where we introduce Microsoft 365 Developer Program in Teams Toolkit, according to the latest update of Microsoft 365 Developer Program. For more details of the updates, see [announcement - Latest updates to the Microsoft 365 Developer Program](https://devblogs.microsoft.com/microsoft365dev/stay-ahead-of-the-game-with-the-latest-updates-to-the-microsoft-365-developer-program/).
```

Comment:
suggest to remove "where we introduce Microsoft 365 Developer Program in Teams Toolkit,"

and "- Latest updates to the Microsoft 365 Developer Program"